### DISCUSSION TITLE
Cursos Presenciais de qualidade

### DISCUSSION THREAD

**Discussion by andrelacerdasp** [frontendbr/forum#1896] [Discussão]
Olá pessoal,

Estou pensando em mudar de carreira, sempre tive vontade de aprender a programar, e já estou pegando algumas aulas no Youtube.

Estou procurando um curso presencial para ser desenvolvedor Full Stack, pois EaD pra mim não rola. Eu notei que existem diversos tipos de curso, intensivos de 1 mês até outros de 6 meses. Valores entre R$3.000,00 até 25.000,00. Pesquisei no Google, e achei empresas como Impacta, Training, Let's Code, Caelum e outras porém não conheço nenhuma delas.

Eu queria saber da comunidade se alguém já se formou em algum desses cursos presenciais, e se poderiam me indicar algum com boas indicações em São Paulo, lembrando que sou um absoluto iniciante nessa área.

Obrigado pela atenção!

**Comment by jocelinoFG017** [comment]
Olá, eu sou discente em Sistemas de Informação(Tô no 6º semestre), mas programava no ensino médio(curso técnico) antes de entrar na faculdade, enfim. Vou tentar te dizer o meu ponto de vista.

Cara curso EaD pra mim não rola também, presencial é muito melhor ainda mais em um curso de programação em que nada se aprende da noite pro dia. A arte de programar requer tempo, dedicação e muita paciência, porque as vezes seu código tem umas 1000 linhas e o erro ta em uma linha sem ; (ponto e virgula), em um parenteses() ou uma chave{} não fechada. Claro dependendo da linguagem que você escolher talvez nem tenha ; como é o caso da linguagem python. Infelizmente não tenho nenhum curso presencial pra te indicar em SP porque não sou de SP. Mas o primeiro passo é pesquisar nas universidades locais um curso de tecnologia da informação como Ciência da Computação, Sistemas de Informação ou algum outro semelhante que duram em média 4 anos. Para concluir seja forte e não desista programar é muito massa e "...não se apaixone por uma linguagem especifica pois as tecnologias vivem mudando... ". 

PS: Espero ter ajudado nem que seja um pouquinho (0.2%) 👍🏽  .
